src/server/scripts/EasternKingdoms/ZulGurub/boss_thekal.cpp

```diff
@@ -15,43 +15,48 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "SharedDefines.h"
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
 #include "TaskScheduler.h"
 #include "zulgurub.h"
 
 enum Says
 {
-    SAY_AGGRO                 = 0,
-    SAY_DEATH                 = 1,
+    BOSS_SAY_AGGRO                 = 0,
+    BOSS_SAY_DEATH                 = 1,
+    BOSS_EMOTE_DIES                = 2,
```

Comment:
remove boss prefix

Reply:
Is this a good summary of the logic you're looking for?

- if the ENUM value refers to the boss the file is named for, no prefix - assume its for that boss
- if the ENUM value refers to any other creature, prefix with the type of value, followed by an indicator of what creature it goes with (i.e. `EMOTE_ZEALOT_AGGRO` or whatever)

I have updated as you indicated, but want to know for going forward.

Reply:
I think voice lines should match the comments in the DB generally. The enum itself usually refers to the creature already in the name of the enum afaik. There might be quite a few outliers in this case, but generally we want it to be consistent like that

Reply:
See my latest namespace solution and see if that resolves any enum naming concerns.